doc/developer-guides/kernel-parameters.rst

```diff
@@ -0,0 +1,380 @@
+.. _kernel-parameters:
+
+ACRN Kernel Parameters
+######################
+
+ACRN supports flexible configurations so customers can configure SOS and
+UOS behavior easily, and according to their requirements.  This document
+introduces kernel command line options used to configure and boot VMs.
+
+
+.. list-table::
+   :header-rows: 1
+   :widths: 10,10,50,30
+
+   * - Parameter
+     - Used in SOS or UOS
+     - Description
+     - Usage example
+
+   * - module_blacklist
+     - SOS
+     - A comma-separated list of modules that should not be loaded. Useful for debugging problem modules.
```

Comment:
This reads a bit strange to me: "Useful for debugging problem modules." Can we improve it? Maybe something like "Useful to debug or work around issues related to specific modules"